### PR TITLE
Sva 219 welcome info only show on first use of the app

### PR DESCRIPTION
**Pull request by DougieWougie**
- [Description](#description)
  - [Type of change](#type-of-change)
- [Checklist](#checklist)

# Description
- Added option to show/not show welcome screen
- Added reducers and amended store
- Buttons change colour to indicate state
- **Note the error at the top of the screen is a separate bug (#93)**

## Type of change

Please delete options that are not relevant
- [] Bug fix (non-breaking change which fixes an issue)
- [X] New Feature (non-breaking change which adds functionality)
- [] Breaking change(fix or feature that would cause existing functionality to not work as expected)
- [] This change requires a documentation update

# Checklist

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [] I have made corresponding changes to the documentation (if required)
- [X] My changes generate no new warnings
- [] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by joefhall — CHANGES_REQUESTED** [review]
Nice work @DougieWougie!  Works well -- and definitely a handy feature to have while we're developing the app 🙂  Just spotted a few small things.

File: app/src/Containers/WelcomeContainer.tsx [hunk 1]
```diff
@@ -1,12 +1,13 @@
 import React, {useState, useRef} from 'react'
 import { useTheme } from '@/Hooks'
-import {Dimensions, I18nManager, Platform, SafeAreaView} from 'react-native'
+import {Dimensions, I18nManager, Platform, SafeAreaView, StyleSheet, View} from 'react-native'
```
[the remark this answers is not in the document]
Comment (DougieWougie) [reply app/src/Containers/WelcomeContainer.tsx:3]: Good spot. I need to change by default VS Code colour scheme - the bluish grey looks the same!

**Comment by DougieWougie** [comment]
> Nice work @DougieWougie! Works well -- and definitely a handy feature to have while we're developing the app 🙂 Just spotted a few small things.

Thanks @joefhall - appreciated!

**Review by joefhall — APPROVED** [review]
Thanks @DougieWougie 🙌

**Review by g-dunlop — APPROVED** [review]
Works perfectly for me @DougieWougie. Nice one!

**Pull request merged**